### PR TITLE
#966 Added isLoggingEnabled to EventBus

### PR DESCRIPTION
**Pull request by wolches**
Hello!

Not sure if my code are breaking any accepted practices or code conventions.
Is it ok?

### CODE REVIEW

File: fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt [hunk 1]
```diff
@@ -24,6 +24,8 @@ class EventBus {
 
     private val eventHandlers = Group()
 
+    private var loggingEnabled = true
```
Comment (AlmasB) [inline fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt]: 1. Please remove `private`, Kotlin will auto-generate `getter/setter` pair, so the two new functions below are no longer needed.
2. Please rename variable name to `isLoggingEnabled` to preserve consistency in the codebase (all boolean variables begin with `is`).

File: fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt [hunk 2]
```diff
@@ -48,8 +50,18 @@ class EventBus {
...
     fun fireEvent(event: Event) {
-        log.debug("Firing event: $event")
+        if (loggingEnabled) {
+            log.debug("Firing event: $event")
+        }
 
         eventHandlers.fireEvent(event)
     }
+
+    fun isLoggingEnabled(): Boolean {
```
Comment (AlmasB) [inline fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt]: These two functions are not needed after the above change (removing `private`) is made.

**Review by AlmasB — CHANGES_REQUESTED** [review]
Looks good, please consider the comments attached.

File: fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt (hunk 2 above)
Comment (wolches) [reply fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt]: Fixed

File: fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt (hunk 1 above)
Comment (wolches) [reply fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt]: Fixed

**Review by AlmasB — APPROVED** [review]
Great, many thanks!

**Pull request merged**